Still doesn't build with kwin 5.19+

After https://github.com/linuxdeepin/dde-kwin/commit/7bb00d244b95b0e2193556d5ff9e73dd29a54d4a, dde-kwin still doesn't build with kwin 5.19:

```
plugins/kwineffects/blur/blur.cpp: In constructor ‘BlurEffect::BlurEffect(QObject*, const QVariantList&)’:
plugins/kwineffects/blur/blur.cpp:65:28: error: ‘class KWaylandServer::BlurManagerInterface’ has no member named ‘create’
   65 |             m_blurManager->create();
      |                            ^~~~~~
```

I removed this line in my previous patch. Not sure what is the right fix, though.

--- a/plugins/kwineffects/blur/blur.cpp
+++ b/plugins/kwineffects/blur/blur.cpp
@@ -60,7 +60,9 @@ BlurEffect::BlurEffect(QObject *, const QVariantList &)
         KWaylandServer::Display *display = effects->waylandDisplay();
         if (display) {
             m_blurManager = display->createBlurManager(this);
+#if defined(KWIN_VERSION) && KWIN_VERSION <= KWIN_VERSION_CHECK(5, 18, 90, 0)
             m_blurManager->create();
+#endif
         }
     } else {
         net_wm_blur_region = 0;

